graylog2-server/src/main/java/org/graylog/events/processor/EventProcessorConfig.java

```diff
@@ -68,5 +69,10 @@ public String type() {
         public ValidationResult validate() {
             throw new UnsupportedOperationException();
         }
+
+        @Override
+        public Object toContentPackEntity() {
+            return null;
```

Comment:
Shouldn't this return something?

Reply:
I throw there an UnsupportedOperationException which is what the other functions do as well. If you would rather have a `Optional.empty()` I can do that as well. But at some point I have to raise an exception, because I can't go to defined behavior with a `FallbackConfig`